tests/generative/variable_strategies.py

```diff
@@ -273,6 +274,12 @@ def variable(
     to_first_of_year = qm_builds(Function.ToFirstOfYear, date_series)
     to_first_of_month = qm_builds(Function.ToFirstOfMonth, date_series)
 
+    case = qm_builds(
+        Case,
+        st.dictionaries(series, series, min_size=1, max_size=3),
```

Comment:
Initially I had `max_size=5`, but with 20K examples it still didn't manage to generate one that used `Case` - presumably because there were too many possible non-homogenous inputs that got thrown away.  With a max of 3, it would work with 10K examples.

Reply:
Yeah, that makes sense: it's not even that you're trying to get 5-of-a-kind by randomly dealing cards; you need 5 homogeneous-typed values _and_ 5 boolean-typed keys, so the changes get tiny very quickly as you increase N.

This is the sort of thing that the alternative approach Ben is working on will hopefully solve for us.